documentation/basic/basic.md

```diff
@@ -174,9 +174,12 @@ int main(void)
 
 ## RT-Thread Program Memory Distribution
 
-The general MCU contains storage space that includes: on-chip Flash and on-chip RAM, RAM is equivalent to memory, and Flash is equivalent to hard disk. The compiler classifies a program into several parts, which are stored in different memory areas of the MCU.
+The general MCU contains storage space that includes: 
+on-chip Flash 
+on-chip RAM. 
+RAM is equivalent to memory, and Flash is comparable to a hard disk. The compiler classifies a program into several parts stored in different memory areas of the MCU.
```

Comment:
Just integrate in one paragraph. Don't need to start a new line.

Reply:
Removed errors and integrated it into one paragraph.